workflow-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -0,0 +1,10 @@
+def branches = [:]
+for (int i = 0; i < 4; i++) {
+  def which = "${i}" 
```

Comment:
My question is why a separate variable when you just use it once? Why not just put `"${i}"` on line 6 instead of `which`? Makes things a little more readable that way, IMO.